src/braket/default_simulator/density_matrix_simulator.py

```diff
@@ -156,6 +156,12 @@ def properties(self) -> GateModelSimulatorDeviceCapabilities:
                             {"name": "Probability", "minShots": 0, "maxShots": max_shots},
                             {"name": "DensityMatrix", "minShots": 0, "maxShots": 0},
                         ],
+                        "supportPhysicalQubits": False,
```

Comment:
Is this a meaningful restriction for the simulators ?

Reply:
This flag is false because the $n syntax isn't supported for referencing qubits on this device. We decided this made sense and is a one-way door in that we can always add support for this at a later point if we have a use case.